packages/kbn-investigation-shared/src/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export * from './rest_specs';
```

Comment:
@kdelemme Ι see you export here only from rest_specs. Does it make sense to export also from schema folder here as well?

```
export * from './schema'
export * from './rest_specs';
```